src/mutations/bookings.ts

```diff
@@ -60,13 +63,8 @@ const updateBooking = async (parent, data, ctx): Promise<Booking> => {
   });
 };
 
-const deleteBooking = async (parent, { id }, ctx): Promise<Booking> => {
-  const currentUser: User | null = await resolveUserUsingJWT(ctx);
-  assert.notStrictEqual(currentUser.id, null, "");
-  const bookingUser: Booking = await ctx.prisma.booking({ id }).user();
-  console.log(currentUser.id, bookingUser.id);
-  assert.strictEqual(currentUser.id, bookingUser.id, "User is not allowed");
-
```

Comment:
this seems to be removed in this branch? is this intended?

Reply:
I doubt

Reply:
I think there has been a issue during my rebase